src/platform/forms-system/src/js/utilities/data/findDuplicateIndexes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Find duplicates in an array of objects and return an array of indexes for the
+ *  duplicates only
+ * @param {Object} formData - Data array (ArrayField data)
+ * @param {String} dataKey - key of data of interest to find duplicates
+ * @return {Array} - Array of indexes of the duplicates only, not the first
+ *  entry
+ * @example
+ * findDuplicateIndexes([{x:'one'},{x:'two'},{x:'one'},{x:'two'}], 'x')
+ * // => [2, 3]
+ */
+export default function(formData = [], dataKey = '') {
+  if (!dataKey || formData?.length === 0) {
```

Comment:
```suggestion
  if (!dataKey || formData.length === 0) {
```
Shouldn't need this with the default arg, right? Transpiling optional chaining adds a little bit of extra code to the bundle sent to the client, so if it's not needed we shouldn't do it.